### PR TITLE
Use additional search handler for distributed searchs to make spellchecker work.

### PR DESCRIPTION
**Pull request by maethu**
🚧 hold on, currently testing.

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -433,6 +433,16 @@ Example ``development.cfg``:
...
 
+Custom standard solr config templates
++++++++++++++++++++++++++++++++++++++
+
+You can choose the solr config template with the ``solr-template-name`` variable in solr-settings.
+
+Currently there are two solr4 templlates:
+
+- 01_original.xml.tmpl --> Copy of solr condig template from the recipe.
+- 02_spellchecker.xml.tmpl (Default) --> Includes spellcher support for multi shards setup.
```
Comment (jone) [inline README.rst]: `spellcher`

File: README.rst [hunk 2]
```diff
@@ -433,6 +433,16 @@ Example ``development.cfg``:
         https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/plone-development-solr.cfg
         solr.cfg
 
+Custom standard solr config templates
++++++++++++++++++++++++++++++++++++++
+
+You can choose the solr config template with the ``solr-template-name`` variable in solr-settings.
```
Comment (jone) [inline README.rst]: I propose to add a hint about where those files are stored.

File: solr-base.cfg [hunk 3]
```diff
@@ -57,7 +66,6 @@ java_opts =
 cores = ${solr-settings:solr-cores}
 default-core-name = ${solr-settings:solr-default-core}
 
-
```
Comment (jone) [inline solr-base.cfg]: ✂️

**Comment by lukasgraf** [comment]
Hallo

File: solr.cfg [hunk 4]
```diff
@@ -1,5 +1,5 @@
 [buildout]
-extends = https://raw.githubusercontent.com/4teamwork/ftw-buildouts/master/solr-base.cfg
+extends = https://raw.githubusercontent.com/4teamwork/ftw-buildouts/mle-solr-suggestion/solr-base.cfg
```
Comment (jone) [inline solr.cfg]: 🚧 To be reverted before merge

File: README.rst [hunk 5]
```diff
@@ -433,6 +433,16 @@ Example ``development.cfg``:
...
+
+Currently there are two solr4 templlates:
```
Comment (jone) [inline README.rst]: `templlates`

File: README.rst [hunk 6]
```diff
@@ -433,6 +433,16 @@ Example ``development.cfg``:
...
+
+- 01_original.xml.tmpl --> Copy of solr condig template from the recipe.
```
Comment (jone) [inline README.rst]: `condig`

File: README.rst (hunk 1 above)
Comment (jone) [inline README.rst]: `multi shards setup` => `multi-shard setup`

**Comment by jone** [comment]
@maethu close?

**Comment by maethu** [comment]
close

**Pull request closed without merging**